2_accept_offer.py

```diff
@@ -93,7 +93,11 @@ def process_offer(offer, audit):
     + locale.format("%.8f", Decimal(offer_currency_quantity) / COIN, True) + " " + offer_currency_code + " for " \
     + locale.format("%.8f", Decimal(ask_currency_quantity) / COIN, True) + " " + ask_currency_code
 
-  # TODO: Prompt the user for whether the trade is acceptable
+  answer = raw_input("Do you want to continue with the trade? 'Y' to continue, 'N' to quit.  ").lower()
+  while answer not in ['y', 'yes', 'n', 'no']:
+    answer = raw_input("Not recognised. 'Y' to continue, 'N' to quit.  ").lower()
+  if answer != 'y' or answer != 'yes':
```

Comment:
Shouldn't this be "and"? As is, it wants it to be both 'y' and 'yes' in order to not exit.